### DISCUSSION TITLE
Change the WAClient logo please.

### DISCUSSION THREAD

**Discussion by albakhtari** [cstayyab/WALC#191] [Feature Requests]
It looks awful, I think you should change it to this:

![WAClient](https://user-images.githubusercontent.com/79008923/167356862-7fa949e9-8a61-4ef5-8261-551b2d8507c4.png)

**Comment by daffamumtaz2361** [comment]
You'd need to ask @cstayyab for that, I don't have access to do that

**Comment by albakhtari** [comment]
@cstayyab, when you have time, can you please change the WAClient, بہت شکریہ :grin:

**Reply by albakhtari** [reply]
You know what, I'll just email him.
